config/locales/idv/en.yml

```diff
@@ -39,16 +39,16 @@ en:
         Please check that the personal and financial information you provided is accurate.
         To learn more about the information we need from you and how it’s used to verify your identity,
         please see our troubleshooting page.
-      activated: >
+      activated_html: >
         Your identity has been verified. If you need to change your verified information,
-        please %{link}.
+        please <a href=%{url}>%{link_text}</a>.
```

Comment:
would it be easier to generate the link inside the slim?

```yml
activated_html: >
  ... %{link}
```

```slim
t('activated_html', link: link_to(url, t('link_text'))
```

A few reasons, I feel somewhat strongly about this (like 7/10)

1. Don't have to worry about escaping things inside the HTML (ex there are no quotes around `url` in this template
2. Easier to change the link (ex if we want `target="_blank"` all we have to change is the `.slim`)
3. I've had bad experiences in the past sending HTML inside strings through translation, people have made small bugs (swapping straight quotes for curlies, removing end tags) and it's broken our apps in production